lib/management/configuration.js

```diff
@@ -158,8 +167,8 @@ function patchConfiguration(newConf, log, cb) {
                             bucketName: l.details.bucketName,
                             mpuBucketName: l.details.mpuBucketName,
                             bucketMatch: l.details.bucketMatch,
-                            gcpEndpoint: l.details.endpoint ||
-                                'storage.googleapis.com',
+                            gcpEndpoint: gcpEndpoint,
+                            https: https,
```

Comment:
nothing major, `https: https` can be written as just `https` here

Reply:
That's cool.